pkg/kubectl/cmd/attach.go

```diff
@@ -265,75 +279,63 @@ func (p *AttachOptions) Run() error {
 			sizeQueue = t.MonitorSize(&sizePlusOne, size)
 		}
 
-		// unset p.Err if it was previously set because both stdout and stderr go over p.Out when tty is
+		// unset o.Err if it was previously set because both stdout and stderr go over o.Out when tty is
 		// true
-		p.ErrOut = nil
+		o.ErrOut = nil
```

Comment:
I'm hesitant about this action. We're modifying global `IOStreams` object. I'd prefer we make a local copy of `ErrOut` and pass that to `AttachFunc` as needed.